README.md

```diff
@@ -6,3 +6,12 @@
 * The first front end for users should be a web page, with a form accepting a text or a file with the course contents.
 * The return should be presented nicely in the webpage and preferably be easy to copy.
 * An optional feature is delivery of a downloadable generated .pdf out from the page.
+
+
+# server.js
+To run server.js
+1. Go to project directory in terminal
+2. run command: `npm install express`
```

Comment:
Is it not better to simply run
`npm install`
which will install all packages listed in the [package.json](package.json) file?

Reply:
Honestly didn't know / think that would work. But I tested it and you are correct. It's now changed.